### ISSUE TITLE
Gender Identity & Expression in the Code of Conduct

### ISSUE THREAD

**Issue by Charlotteis** [nodejs/inclusivity#82]
## Purpose

I'd really like to learn Node.js this year and to do that I usually try and get involved in a part of a community. Before I try and take part in a community, I generally look at the Code of Conduct first.

I noted that the [Code of Conduct](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md) mentions 'gender' but that doesn't really feel like it includes [non-binary](http://gender.wikia.com/wiki/Non-binary) or [agender](http://gender.wikia.com/wiki/Agender) folk.

I realise the CoC says 'similar personal characteristic', but it feels like it'd be good to have things explicitly said rather than assumed. The [Contributor Covenant](http://contributor-covenant.org/) has a lot of really useful things in it too (like age, technology choices, lack of religion), which we could consider co-opting things from.

## Focus

This dicussion is specifically about including 'gender identity and expression' into the Node.js Code of Conduct. Discussions that don't consider including gender identity into the Code of Conduct should be considered off topic.

Some off topic examples include, but are not limited to:
- Is gender a social construct?
- Do we really think that people should even have genders any more?
- The validity of other sections of the Code of Conduct
- The usefulness or necessity of Codes of Conduct in general

Have a good day :sparkles:

**Comment by ashleygwilliams** [comment]
hey @Charlotteis thanks for filing! this is great and we'd love to help you with this effort.

the first step will be to merge our own CoC, which i plan on finally doing today or tomorrow. plz share your thoughts!

next will be writing a proposal via PR, would you be available to join us for our meeting next week on Thursday at 10am PST?

**Comment by nebrius** [comment]
Thanks for writing in! A couple links that may be of interest to you:
Code of Conduct PR for this working group: https://github.com/nodejs/inclusivity/pull/22
Issue structure PR for this working group: https://github.com/nodejs/inclusivity/pull/71

**Comment by ashleygwilliams** [comment]
oh good call @nebrius i was gonna mention that because @Charlotteis has good Opinions:tm: about such things

**Comment by ashleygwilliams** [comment]
also #65 :hint::hint::nudge:nudge:

**Comment by Charlotteis** [comment]
Cool, I see that the WG CoC would include gender identity, which is awesome.

I'm going to go ahead and make a coffee and read ~all of the things ever~ in this WG. I do love to have (helpful?) opinions!

Hi @nebrius :wave: , thanks so much for those links.

As for 10am PST.... I'm usually on the train home from work at that time but I will certainly see what I can do!

**Comment by nebrius** [comment]
alright @ashleygwilliams I'm going I'm going (to #65). _shambles over_ 😛

**Comment by ashleygwilliams** [comment]
lol i was meaning to suggest to @Charlotteis that she czech it out to join! but yeah also lol it'd be great to ship a 1.0 version :dolphin:

**Comment by nebrius** [comment]
@Charlotteis if you've got time to take a look, it'd be great to get your feedback on https://github.com/nodejs/inclusivity/issues/74 as well

**Comment by Charlotteis** [comment]
Slowly going to go through everything.The WG README mentions 'gender and sex', should I open a new issue to reword this to 'gender identity (and expression) and sex'?

**Comment by Charlotteis** [comment]
Rust now includes this in their CoC (https://github.com/rust-lang/rust-www/pull/270), which may be useful for our discussion here (only, as mentioned before, node seems to specifically mention how they used the Rust CoC) :sparkles:

**Comment by ashleygwilliams** [comment]
A++

**Comment by Charlotteis** [comment]
@nebrius I took a look at your Issue structure and rewrote my issue to adhere to it. Great work! I've also gone through all of the other issues/PR's linked and have commented on them where I felt it was needed :)

**Comment by nebrius** [comment]
Wow, thanks for all the feedback @Charlotteis, this is awesome! :)

**Comment by Charlotteis** [comment]
Hai! Rust just landed this: https://github.com/rust-lang/rust-www/pull/288
I bring it up again because I wonder if the scope of this issue could include similar personal characteristics? I'm happy to keep this to be just about 'gender identity' as I'm sure that this WG will work hard to keep the CoC up to date in general and it's probably going to be an ongoing process. But, it's worth considering broadening scope a little. Just something we can think about!

**Comment by zkat** [comment]
Wait, can we just file a PR against nodejs/node with this change? I will +1 this patch myself, although it may end up landing in `tsc-agenda` just because someone may just really want that to happen.

If this seems good, would @Charlotteis be interested in cooking up that PR and submitting it? If you're not comfortable doing so (because, understandably, there is always a chance for backlash), I would gladly volunteer for submitting it myself.

**Comment by nebrius** [comment]
+1 to just filing a PR against nodejs/node. I kinda have the feeling this should be pretty uncontroversial (to collaborators), since it doesn't change enforcement policies or anything _and_ it was landed in Rust.

**Comment by Fishrock123** [comment]
Two things about this:

I interpret this as a doc change and not a policy change. Being both a TSC (moderator) and CTC (policy owner) member.

I'd prefer if someone besides myself was comfortable PRing it, but I will do it if necessary.

**Comment by Fishrock123** [comment]
> _and_ it was landed in Rust.

Note: just because we used the Rust CoC originally makes no promises that we will follow it, but pointing this out won't hurt either. :)

**Comment by Charlotteis** [comment]
So I would *love* to have my name up there in the contributors, and I'm in half a mind to do it myself. However, I've already had some crappy experiences lately with regards to CoC changes (and lets not forget the current uproar in the community this week with this stuff) and would appreciate someone actually known with the Node community and known by Core to make this PR if that's possible. Thanks so much for all of this, I'm a very happy computer right now :computer: :sparkles:

**Comment by zkat** [comment]
I'll cook one up, then. Stand by.

**Comment by TheAlphaNerd** [comment]
🎉🎉🎉🎉🎉

**Comment by Charlotteis** [comment]
![thumbsup](https://cloud.githubusercontent.com/assets/542140/12524135/e84ae7b2-c152-11e5-80aa-3c2b6d113f18.gif)

a special thank you .gif for everyone's help with this <3 <3
